contract_manager/package.json

```diff
@@ -41,6 +41,9 @@
     "aptos": "^1.5.0",
     "axios": "^0.24.0",
     "bs58": "^5.0.0",
+    "extract-files": "^13.0.0",
+    "fuels": "^0.89.2",
+    "ramda": "^0.30.1",
```

Comment:
`extract-files` and `ramda` was needed by `fuels`

Reply:
what do you mean? they are peer dependencies for some functionalities or :?